libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/FusedLocationMocker.kt

```diff
@@ -7,9 +7,14 @@ import android.util.Log
 import com.google.android.gms.location.FusedLocationProviderClient
 
 internal class FusedLocationMocker(
-    appContext: Context
+    appContext: Context,
+    private val mockProviderName: String = DEFAULT_PROVIDER_NAME
```

Comment:
Why can this change? Fused is fused, it can't be anything else.

Reply:
I wanted to follow the convention of the other provider. Perhaps we'd want to filter samples by name in the future and assert them? Just a thought though, the change is just to keep parity.

Reply:
I think that if we use `FusedLocationMocker`, we mean "fused" provider. It should be incapsulated in the class itself. Parity is not required here because provider can only change for system provider.
Well, if we wanted to sanity check google play location library (what if it gives us locations with weird providers) we might use it of course.